scalyr_agent/scalyr_client.py

```diff
@@ -194,7 +198,7 @@ def ping(self):
         @return:  The status message returned by the server.
         @rtype:
         """
-        return self.send(self.add_events_request())[0]
+        return self.send(self.add_events_request(disable_new_addevents_format=self.__disable_new_addevents_format))[0]
```

Comment:
You shouldn't need to pass this argument to `add_events_request` since this object already has that value.  See comment below.